docs/contributing.md

```diff
@@ -23,9 +23,9 @@ We're glad you want to contribute to Reliability Kit!
 
 In order to contribute and make changes to this repository, you'll need some software installed:
 
-  * [Node.js](https://nodejs.dev/) v14 or higher
+  * [Node.js](https://nodejs.dev) on a version defined in `package.json`: `engines.node`
 
-  * [npm](https://www.npmjs.com/) v7 or higher (usually bundled with Node.js)
+  * [npm](https://www.npmjs.com) on a version defined in `package.json`: `engines.npm` (usually bundled with Node.js)
```

Comment:
I always find the documentation is usually the place that forgets to get updated as part of a Node upgrade as that it is more update-proof for the docs to simply referencing somewhere else in the codebase where the Node version depends on being updated.

Reply:
Good shout, one less place to update 👍